finality-provider/service/fp_instance.go

```diff
@@ -669,6 +709,9 @@ func (fp *FinalityProviderInstance) retryCommitPubRandUntilMaxRetry(targetBlockH
 // CommitPubRand generates a list of Schnorr rand pairs,
 // commits the public randomness for the managed finality providers,
 // and save the randomness pair to DB
+// Note:
+// - if there is no pubrand committed before, it will start from the targetBlockHeight
```

Comment:
in the code, it seems it starts from `0` but either way it should start from a future height considering btc timestamping latency. See https://github.com/babylonlabs-io/finality-provider/blob/main/docs/commit-pub-rand.md#determining-start-height